src/foxops/__main__.py

```diff
@@ -70,9 +70,13 @@ async def startup():
 
     # Add static content
     for frontend_dir in FRONTEND_SUBDIRS:
+        path = settings.frontend_dist_dir / frontend_dir
+        if not path.exists():
```

Comment:
Let's at least log a warning:

```suggestion
        if not path.exists():
            logger.warning(f"The static asset path at {path} does not exist, skipping ...")
```